### PR TITLE
Add message for no pull requests

### LINKED ISSUE #16: Empty result for user without merge requests
Problem is that there's any information messages for user without merge requests. Try to enter `test` on the [demo](https://pullrequests.surge.sh/test)

### PR DESCRIPTION
**Pull request by uLan08**
Fixes #16

### CODE REVIEW

File: src/components/User.js [hunk 1]
```diff
@@ -26,13 +36,21 @@ class User extends Component {
...
     console.log(this.props);
     if (this.state.loading) { return (<Load />); };
-    return (
-      <Row>
-        {this.state.data.items.map((pullrequest, idx) => {
-          return <PullRequest data={pullrequest} key={idx} />;
-        })}
-      </Row>
-    );
+    const { items } = this.state.data
```
Comment (ducthienbui97) [inline src/components/User.js]: Codacy is complaining about this line.
https://www.codacy.com/app/ducthienbui97/pullrequests/pullRequest?prid=957551

File: src/components/User.js [hunk 2]
```diff
@@ -1,8 +1,18 @@
 import React, { Component } from "react";
-import { Row } from "react-bootstrap";
+import { Row, Col } from "react-bootstrap";
 import Axios from "axios";
 import PullRequest from "./PullRequest";
 import Load from "./Load";
+
+const style = {
```
Comment (ducthienbui97) [inline src/components/User.js]: Can you use different file for css?

**Review by ducthienbui97 — CHANGES_REQUESTED** [review]

**Comment by nghduc97** [comment]
Missing `;` at the end of line.

File: src/components/User.js (hunk 2 above)
Comment (uLan08) [reply src/components/User.js]: Where would be a good way to place them? Just in the components folder? Or we separate them into a styles folder?

Comment (uLan08) [reply src/components/User.js]: I forgot that there is a assets folder. I'll put them there.

**Review by ducthienbui97 — APPROVED** [review]

**Comment by ducthienbui97** [comment]
@uLan08  Thank you for your contribution 👍 .

**Comment by uLan08** [comment]
@ducthienbui97 Thank you for the opportunity as well. This lets me practice my react skills as I am mainly a Node.js developer.

**Pull request merged**